public/static/docs/command-reference/import.md

```diff
@@ -9,6 +9,10 @@ import.
 > See also `dvc get`, that corresponds to the first step this command performs
 > (just download the data).
 
+<!-- Separate MD quote: -->
+
+> See also our `dvc.api.open` Python API function.
```

Comment:
Too many notes, merge into one and/or move into Description?

For example the first note, could be eliminated with `Same as dvc get, but also creates a DVC-file ...`?

Reply:
Simplified and moved around these and other notes in 0a725e8e.